Utilities/BEType.h

```diff
@@ -322,6 +323,36 @@ union alignas(16) v128
 		return fromD(_mm_cmpeq_pd(left.vd, right.vd));
 	}
 
+	static inline bool use_fma = false;
```

Comment:
Why just don't set it to ```utils::has_fma3()``` right there, like it's done in other places.

Reply:
It will have an initializer, slower to compile